gsoc_contributors_guide.md

```diff
@@ -0,0 +1,33 @@
+# GSoC Contributor's Guide
+
+The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/) participants to help us create the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the the data exploration and discovery process. This project will go hand-in-hand with ongoing development of the [echopype](https://github.com/OSOceanAcoustics/echopype) package that handles the standardization, pre-processing, and organization of these data. We aim for creating ocean sonar data visualization functionalities based on the [HoloViz](https://github.com/holoviz/holoviz) suite of tools, and developing widgets that can be flexibly combined in [Panel](https://panel.holoviz.org/) or used in Jupyter notebooks for interactive data visuzliation.
```

Comment:
Visualization is misspelled.

Reply:
- Per Don's prior comment: "cloud-hosted" > "cloud-native"
- "the the" > "the"
- https://github.com/holoviz/holoviz > https://holoviz.org/
- "suite of tools" > "suite of Python tools"
- "flexibly combined in [Panel](https://panel.holoviz.org/)" > "flexibly combined in [Panel](https://panel.holoviz.org/) dashboards"